magda-sleuther-duplicate-records/src/test/urlsFromDataSet.ts

```diff
@@ -0,0 +1,13 @@
+import { Record } from "@magda/typescript-common/dist/generated/registry/api";
```

Comment:
Once again, move this into typescript-common and refer to it from both here and the broken link sleuther rather than copying it :)